### PR TITLE
fix: Removed use of origional fs to allow extension to work remotely

### LINKED ISSUE #561: SVN doesn't work with Remote-SSH extension
- VSCode Version: 1.34.0-insider
- OS Version: Windows 10 / RHEL 7
- Extension Version: 1.48.1
- System Language: English (US)
- SVN Version: 1.9.7

## Issue ##

I am using the new Remote-SSH extension. When I try to use the SVN extension with it, it reports that "There is no data provider registered that can provide view data".

I thought that it may be that I have svn installed in /opt/subversion/bin/svn, but I tried setting svn.path to that path, and it still doesn't work.

I also tried setting it to my TortoiseSVN path, in case it needs to use the executable on the Windows side for some reason. That doesn't work either.

In the developer's console, there is an error "Cannot find module 'original-fs'". I don't know if that is related.

Note that Git works fine.

## Steps to Reproduce ##

1. Install the Remote-SSH extension.
2. Connect to a Linux system
3. Attempt to use the SVN extension after opening a path that is using SVN.

## SVN Output ##
None - no option in the dropdown.

## Screenshots ##

image

### PR DESCRIPTION
**Pull request by JohnstonCode**
fixes #561

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -166,6 +166,9 @@ export async function activate(context: ExtensionContext) {
     new Disposable(() => Disposable.from(...disposables).dispose())
   );
 
+  //See https://github.com/JohnstonCode/svn-scm/issues/561
+  (process as any).noAsar = true;
```
Comment (edgardmessias) [inline src/extension.ts]: Do not do it

Comment (edgardmessias) [reply src/extension.ts]: The trick: https://github.com/JohnstonCode/svn-scm/blob/master/src/svn.ts#L10 not work when `noAsar` is true

Comment (JohnstonCode) [reply src/extension.ts]: What deps is that trick bringing in again?

**Comment by edgardmessias** [comment]
Instead of change, add the https://www.npmjs.com/package/original-fs, work like a polyfill

File: src/extension.ts (hunk 1 above)
Comment (edgardmessias) [reply src/extension.ts]: To reduce size of ".vsix", some dependencies are removed, to use the embed module in "VSCode"

**Comment by JohnstonCode** [comment]
How does the polyfill work? I mean i know how it exports fs. But how does it not conflict with the other orgional-fs in electron?

File: src/extension.ts (hunk 1 above)
Comment (edgardmessias) [reply src/extension.ts]: See #409

**Comment by edgardmessias** [comment]
For electron, the module "original-fs" is loaded before files

**Comment by edgardmessias** [comment]
Look: https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders

**Pull request merged**